src/WebJobs.Script.WebHost/Management/LinuxInstanceManager.cs

```diff
@@ -91,19 +70,42 @@ public bool StartAssignment(HostAssignmentContext context)
                 _webHostEnvironment.DelayRequests();
 
                 // start the specialization process in the background
-                Task.Run(async () => await AssignAsync(context));
-
-                return true;
+                return Task.Run(async () =>
```

Comment:
Same here, `Task.Run` is no longer needed. All it will do is force a yield as the work inside it gets posted to the thread pool.